<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>301</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>424</width>
    <height>301</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Порт</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>45</y>
     <width>406</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры подключения</string>
   </property>
   <widget class="QComboBox" name="cmb_baudRates">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28800</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость Пердачи</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_DataBits">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>15</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 bit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>DataBits</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_Parity">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No Parity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even Parity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd Parity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Space Parity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mark Parity</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_stopBits">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>45</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>One Stop</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>One And Half Stop</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Two Stop</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>50</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>StopBits</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>85</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FlowControl</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_flowcontrol">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>85</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No Flow Control</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hardware Control</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Software Control</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QComboBox" name="cmb_ports">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>15</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_update_list_port">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>15</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить </string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>175</y>
     <width>396</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_run_test">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>260</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_stop">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>260</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status_port">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>195</y>
     <width>346</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_save_param">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
